keyboards/40percentclub/gherkin/keymaps/mjt/rules.mk

```diff
@@ -1,28 +1,5 @@
-## not sure where this stuff goes ##
-MCU = atmega32u4
-F_CPU = 16000000
-ARCH = AVR8
-F_USB = $(F_CPU)
-OPT_DEFS += -DBOOTLOADER_SIZE=4096
-
-# Build Options
-#   change to "no" to disable the options, or define them in the Makefile in
-#   the appropriate keymap folder that will get included automatically
-#
-BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration(+1000)
-MOUSEKEY_ENABLE = yes       # Mouse keys(+4700)
-EXTRAKEY_ENABLE = yes       # Audio control and System control(+450)
-CONSOLE_ENABLE = no         # Console for debug(+400)
-COMMAND_ENABLE = yes        # Commands for debug and configuration
-NKRO_ENABLE = yes           # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
-BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
-MIDI_ENABLE = no            # MIDI controls
-AUDIO_ENABLE = yes           # Audio output on port C6
-UNICODE_ENABLE = no         # Unicode
-BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
-RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight. 
-SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
-TAP_DANCE_ENABLE=yes
+AUDIO_ENABLE = yes          # Audio output on port C6
+TAP_DANCE_ENABLE = yes
 
 ifndef QUANTUM_DIR
```

Comment:
This block should be removed, as it's not needed

Reply:
This block still needs to be deleted.

Reply:
The TAP_DANCE_ENABLE and AUDIO_ENABLE flags are required for the keymap.
I kept the minimal amount of overrides from the original whole file copy-paste that would keep the things that were changed.

Reply:
Yes, that's fine, but it's the `ifndef QUANTUM_DIR` stuff that needs to be removed.

Reply:
Ah, fixing now.

Reply:
Done

Reply:
@zvecr for future reference, review comments that are attached to a file are typically against the line above the comment thread.

So this thread is about Line 4 (and subsequent lines in this case). Lines 1-3 are only shown for context, like running `git diff`.